### ISSUE TITLE
nvda alpha most resents not reading anything when entering outlook

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#13810]
### Steps to reproduce:
update nvda to the latest alpha version.
open microsoft outlook.
### Actual behavior:
nvda doesn't read anything, you can press any command on the keyboard and the nvda error repetitive sound is heard.
Even opening the log is almost impossible, nvda takes a long time to respond, it seems like it's busy just logging errors.
This is a regression, in yesterday's alpha everything was working fine.
### Expected behavior:
nvda should work with outlook same before nvda upgrade today.
#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
nvda.exe, NVDA alpha-25614,6f211a13

#### Windows version:
11 22621.160

#### Name and version of other software in use when reproducing the issue:
Microsoft Office 16.0.15225.20204

#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
yes work
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes
log:
[outlook1.txt](https://github.com/nvaccess/nvda/files/8929248/outlook1.txt)
[outlook2.txt](https://github.com/nvaccess/nvda/files/8929251/outlook2.txt)

**Comment by lukaszgo1** [comment]
I've submitted a fix for this in #13811. Could you please test [this build](https://ci.appveyor.com/api/buildjobs/2vdpfp7k25v8sbwd/artifacts/output%2Fnvda_snapshot_pr13811-25622%2C152e31ca.exe)?

**Comment by fernando-jose-silva** [comment]
perfect, this build worked correctly, thank you very much.
